scripts/start

```diff
@@ -63,6 +63,14 @@ echo "Starting karen..."
 echo
 ./karen &
 
+echo "Starting backup monitor..."
+echo
+./scripts/backup/monitor &
+
+echo "Starting decoy backup trigger..."
+echo
+./scripts/backup/decoy-trigger &
+
```

Comment:
I won't even say how much I hate this, and this file altogether 😝

Reply:
We can start moving some of these things out to their own systemd services on Umbrel OS. But for people running custom Umbrel installs it does the job for now.